salt/states/service.py

```diff
@@ -825,7 +826,6 @@ def unmasked(name, runtime=False):
 def mod_watch(name,
               sfun=None,
               sig=None,
-              reload=False,
```

Comment:
I am not following the reasoning for removing this from the list of defined kwargs. Could you help me understand please, @deuscapturus ?

Reply:
@cachedout that was actually me. I removed it because it shadows a built-in. Instead, we pop it out of the kwargs below.